Snippets/trim2.sqlpromptsnippet

```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
I like the idea, but not a fan of '2' as the snippet. Violates clean code principle of making things semantic. 

How about `trimc` ('c' for 'cursor') and `trims` ('s' for 'selected')?